neuralprophet/forecaster.py

```diff
@@ -705,6 +705,9 @@ def fit(self, df, freq="auto", validation_df=None, progress="bar", minimal=False
         # When only one time series is input, self.id_list = ['__df__']
         self.num_trends_modelled = len(self.id_list) if self.config_trend.trend_global_local == "local" else 1
         self.num_seasonalities_modelled = len(self.id_list) if self.config_season.global_local == "local" else 1
+        self.meta_name_bool = (
```

Comment:
Not really sure what `meta_name_bool` stands for, could we come up with some more clear naming? For example `meta_use_df_name` or something similar?

Reply:
I agree, for me its also not really clear what meta_name is referring to.

Reply:
Thanks, I agree!
What about:
- meta_used_in_forward
- model_uses_meta
?

Reply:
What exactly is meta even referring to? Its the name of the local dataset in case global / local modelling is used, correct?

Reply:
Yes, the dataloader has meta in it, including the 'ID' of the local dataset.
When doing global/local modeling we need to know the 'ID' of each sample on the batch.

Reply:
changed it to  `meta_used_in_model` - Didn't add "name" on it as this could be used as well once we have static covariates, for example

Reply:
Thanks for the improvement @alfonsogarciadecorral - I find the `meta` naming of this functionality rather unclear, but that is not related to your work - I'll open a follow up issue on that.